app/components/search_result/mini_bento/layout_component.html.erb

```diff
@@ -49,8 +49,8 @@
         <span class="badge rounded-pill text-bg-secondary" data-blacklight-result-count-target="count"></span>
       </div>
 
-      <div data-controller="blacklight-result-count" data-blacklight-result-count-url-value="<%= lib_guides_path(q: params[:q]) %>">
-        <%= link_to(Settings.LIB_GUIDES.APP_SEARCH_URL % { q: params[:q] }) do %>
+      <div data-controller="blacklight-result-count" data-blacklight-result-count-url-value="<%= bento_search_url('lib_guides') %>">
```

Comment:
can you make a separate Stimulus controller for the bento request?

Reply:
done.